src/main/java/org/opensearch/neuralsearch/processor/normalization/RRFNormalizationTechnique.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.processor.normalization;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Locale;
+import java.util.Set;
+
+import org.apache.commons.lang3.math.NumberUtils;
+import org.apache.lucene.search.ScoreDoc;
+import org.apache.lucene.search.TopDocs;
+import org.opensearch.neuralsearch.processor.CompoundTopDocs;
+
+import lombok.ToString;
+import org.opensearch.neuralsearch.processor.NormalizeScoresDTO;
+
+/**
+ * Abstracts calculation of rank scores for each document returned as part of
+ * reciprocal rank fusion. Rank scores are summed across subqueries in combination classes
+ */
+@ToString(onlyExplicitlyIncluded = true)
+public class RRFNormalizationTechnique implements ScoreNormalizationTechnique {
+    @ToString.Include
```

Comment:
```suggestion
    
```

Reply:
I will do the other way around - will keep @ToString(onlyExplicitlyIncluded = true) and add  @ToString.Include for rankConstant and technique name. By default @ToString(onlyExplicitlyIncluded) is false